.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,208 @@
+      "foo": [
+        "bar",
+        "baz"
+      ],
+      "key": "value"
+    },
+    "data": "",
+    "files": {},
+    "form": {},
+    "headers": {
+      "Accept": "*/*",
+      "Accept-Encoding": "gzip,deflate",
+      "Host": "httpbin.org",
+      "User-Agent": "api (node)/4.2.0",
+      "X-Amzn-Trace-Id": "REDACTED"
+    },
+    "json": null,
+    "method": "GET",
+    "origin": "REDACTED",
+    "url": "https://httpbin.org/anything?key=value&foo=bar&foo=baz&baz=abc"
+  },
+  "rawHeaders": [
+    "Date",
+    "Thu, 24 Mar 2022 02:14:36 GMT",
+    "Content-Type",
+    "application/json",
+    "Content-Length",
+    "510",
+    "Connection",
+    "close",
+    "Server",
+    "gunicorn/19.9.0",
+    "Access-Control-Allow-Origin",
+    "*",
+    "Access-Control-Allow-Credentials",
+    "true"
+  ],
+  "reqheaders": {
+    "accept": [
+      "*/*"
+    ],
+    "accept-encoding": [
+      "gzip,deflate"
+    ],
+    "connection": [
+      "close"
+    ],
+    "host": "httpbin.org"
+  },
+  "responseIsBinary": false
+}
+<<<<<<-- cut here -->>>>>>
+
+{
+  args: { baz: 'abc', foo: [ 'bar', 'baz' ], key: 'value' },
+  data: '',
+  files: {},
+  form: {},
+  headers: {
+    Accept: '*/*',
+    'Accept-Encoding': 'gzip,deflate',
+    Host: 'httpbin.org',
+    'User-Agent': 'api (node)/4.2.0',
+    'X-Amzn-Trace-Id': 'REDACTED'
+  },
+  json: null,
+  method: 'GET',
+  origin: 'REDACTED',
+  url: 'https://httpbin.org/anything?key=value&foo=bar&foo=baz&baz=abc'
+}
+```
+
+Now all you have to do is take the data within the `<<<<<<-- cut here -->>>>>>` block and slim it down to only what you need for your API mock (you can always skip `response`):
+
+```json
+{
+  "scope": "https://httpbin.org:443",
+  "method": "GET",
+  "path": "/anything?key=value&foo=bar&foo=baz&baz=abc",
+  "status": 200
+}
+```
+
+Some things you should be aware of:
+
+* You can leave `response` out of `mock.json` as our test suite automatically mocks out a `response` to use so we can assert that what we mocked as a response was what we got.
+* If your API doesn't have a request body you can leave `body` off.
+* If your API request doesn't use any unique headers (eg. `Authentication` or `Content-Type`), then you can generally leave `reqheaders` off.
+* `status` should always be 200.
+* If you're testing something like `multpart/form-data` requests and need to regex a boundary you can rename `mock.json` to `mock.js` and add your regex in. See the `multipart-*` test for examples of this.
+
+#### output.js
+
+Now that you've done all the hard work for setting up your unit test all that's left is to craft the snippet you're expecting we should generate for your use case. You have a couple options for this:
+
+1. Write it manually.
+2. Run the test suite without one so it fails, and then populate what we thought it should be from the data in `har.json` and `openapi.json` into `output.js`.
+
+After that, and all your tests are passing, then you're good! Thanks for helping out!
```

Comment:
This is super thorough and I learned a ton by just reading this—thanks for doing this!